frontend/src/scenes/settings/project/AutocaptureSettings.tsx

```diff
@@ -17,7 +17,7 @@ export function AutocaptureSettings(): JSX.Element {
     return (
         <>
             <p>
-                Automagically capture front-end interactions like pageviews, clicks, and more when using our web
+                Automagically capture front-end interactions like clicks, submits, and more when using our web
```

Comment:
```suggestion
                Automagically capture frontend events, such as any <code>click</code>, <code>change of input</code>, or submission associated with a <code>button</code>, <code>form</code>, <code>input</code>, <code>select</code>, or <code>textarea</code>, when using our web JavaScript SDK.
```

Reply:
@MarconLP did you test it? feel free to commit if so, I didn't bother to run it because it was text changes only.

Reply:
auto merged was enabled so its merged sry, feel free to make a PR with your changes.